cmake/nuttx_add_application.cmake

```diff
@@ -35,13 +35,14 @@ define_property(
 #
 # Usage: nuttx_add_application( NAME <string> [ PRIORITY <string> ] [ STACKSIZE
 # <string> ] [ COMPILE_FLAGS <list> ] [ INCLUDE_DIRECTORIES <list> ] [ DEPENDS
-# <string> ] [ MODULE <string> ] [ SRCS <list> ] )
```

Comment:
It seems that the style of the comment is wrongly formatted in the check of cmake-format. could we consider using the comment fence to protect this part of the code?

https://github.com/cheshirekow/cmake_format/blob/master/cmakelang/doc/README.rst?plain=1#L239-L250

Reply:
Good idea, I reformatted the dockstring to be human readable.